theano/gpuarray/dnn_fwd.c

```diff
@@ -89,12 +89,35 @@ APPLY_SPECIFIC(conv_fwd)(PyGpuArrayObject *input, PyGpuArrayObject *kerns,
 #endif
 
   if (!reuse_algo) {
+    size_t free;
+
+    int err2 = gpucontext_property(ctx, GA_CTX_PROP_LARGEST_MEMBLOCK, &free);
```

Comment:
ctx -> c-ctx

this cause jenkins to error. There is other error, GA_CTX_PROP_LARGEST_MEMBLOCK isn't availlable.